exercises/matching-brackets/example.jl

```diff
@@ -0,0 +1,33 @@
+function matching_brackets(a)
+	a = split(a, "")
+	b,c = ["(", "[", "{"], [")", "]", "}"]
+	a = filter(i -> (i in b || i in c), a)
+	stack = Array{String}(undef, 0)
+	global d, count, valid = Dict([("(", ")"),(")", "("), ("[", "]"), ("]", "["), ("{", "}"), ("}", "{")]), 0,1
```

Comment:
These are only used within the function body, they don't need to be globals.

Perhaps also move `d` into a separate line to make it easier to read.